distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -602,7 +598,7 @@ type Vector a
         arr = Array.new (self_len + that.length)
         Array.copy self.to_array 0 arr 0 self_len
         Array.copy that.to_array 0 arr self_len that.length
-        Vector_Data arr
+        Vector.from_array arr
```

Comment:
Same as above - unnecessary copy in a primitive operation.

Also regarding our short chat during catchup today together with @jdunkerley - I heard we are not using Array.copy anywhere apart from the builder - but seems that we still are - in these primitive operations. Although indeed, if necessary we should be able to move them to become builtins written in Java if we need to. But I just wanted to ensure we remember about this usage as well.

Reply:
Replaced with `from_polyglot_array` in ce6aa5cddb9e6